lib/matplotlib/rcsetup.py

```diff
@@ -988,7 +988,12 @@ def validate_webagg_address(s):
     raise ValueError("'webagg.address' is not a valid IP address")
 
 
-validate_axes_titlelocation = ValidateInStrings('axes.titlelocation', ['left', 'center', 'right'])
+validate_axes_titlelocation = ValidateInStrings('axes.titlelocation',
+                                                ['left', 'center', 'right'])
+validate_xaxis_labellocation = ValidateInStrings('xaxis.labellocation',
```

Comment:
these should be private

Reply:
As in rename to `_validate_xaxis_labellocation` ?

Reply:
Yes